js/form.js

```diff
@@ -0,0 +1,10 @@
+const form = document.querySelector('form');
+const emailError = document.querySelector('.form-error');
+
+form.addEventListener('submit', (e) => {
+  const email = form.email.value;
+  if (email !== email.toLowerCase()) {
+    e.preventDefault();
+    emailError.innerHTML = `<p class="error">Email should be written in lowercase letters: ${email}</p>`;
+  }
```

Comment:
- [ OPTIONAL ] I think here when you you display the error you should add `${email.toLowerCase()}` to display to the user the template that should use 💪🏻💪🏻